docs/tutorials/getting-started/go.mdx

```diff
@@ -251,6 +253,7 @@ go run main.go
 Revisit the endpoint [http://localhost:8080/hello](http://localhost:8080/hello) and you will see the same value.
 
 <FlagdChangeContent/>
+NOTE: Remember to restart flagd service that running in Docker.
```

Comment:
It shouldn't be necessary to restart flagd. Was this not working for you?

Reply:
Yes, it wasn't. Flagd container does not aware that flags.flagd.json file was updated.